packages/gluestick/src/commands/autoUpgrade.js

```diff
@@ -0,0 +1,8 @@
+/* @flow */
+import type { Context } from '../types';
+
+const autoUpgrade = require('../autoUpgrade/autoUpgrade');
```

Comment:
To follow other commands convention, I would move the `autoUpgrade` directory inside the `commands` folder. WDYT?